binding/web/README.md

```diff
@@ -91,21 +91,37 @@ npx pvbase64 -h
 
 ### Init options
 
-Rhino saves and caches your model file in IndexedDB to be used by Web Assembly. Use a different `customWritePath`
-variable to hold multiple model values and set the `forceWrite` value to true to force re-save the model file.
-Set `processErrorCallback` to handle errors if an error occurs while transcribing.
+Rhino saves and caches your model (`.pv`) and context (`.rhn`) files in the IndexedDB to be used by Web Assembly.
+Use a different `customWritePath` variable to hold multiple model values and set the `forceWrite` value to true to force a re-save of the model file.
 If the model file (`.pv`) changes, `version` should be incremented to force the cached model to be updated.
 
 ```typescript
-// these are default
+const rhinoContext = {
+  publicPath: ${CONTEXT_RELATIVE_PATH}, // or base64: ${CONTEXT_BASE64_STRING},
+  customWritePath: 'custom_context'; // Optional
+  forceWrite: true; // Optional
+  version: '1.0'; // Optional
+}
```

Comment:
if both are included -> base64 will be used should be mentioned, something like:

Either use `base64` or `publicPath` to the model to instantiate Rhino. If both are
set, Rhino will use the `base64` model.